src/ctapipe/irf/cuts/quality_cuts.py

```diff
@@ -0,0 +1,58 @@
+from astropy.table import Table
+
+from ...core import QualityQuery
+from ...core.traits import List, Tuple, Unicode
+
+
+class EventQualitySelection(QualityQuery):
+    """
+    Event pre-selection quality criteria for IRF and DL3 computation with different defaults.
+    """
+
+    quality_criteria = List(
```

Comment:
We can't really allow the user to select events in a  totally arbitrary way when going from DL2→DL3, as the cuts applied in DL2 also define the IRFs, and if you change them, the IRFs are wrong.  So I think the correct thing here is not to allow a general QualityQuery (though maybe we need a rough one to drop missing or non-reconstructed events), but rather to load the cuts that are output from `ctapipe-optimize-event-selection` and apply them to the observed DL2 data.